### ISSUE TITLE
would you please share your paper

### ISSUE THREAD

**Issue by wzl2611** [Kwanss/PCLNet#1]
You have done a greate job.but I canont find you  paper in the internet. Would you please share your paper?  Thank you very  much

**Comment by Kwanss** [comment]
Yes.  I had uploaded my paper to arxiv yesterday and would be available in a few days.  Thx

**Comment by Ecalpal** [comment]
[https://arxiv.org/abs/1907.11628](url) @wzl2611 @Kwanss

**Comment by Ecalpal** [comment]
@Kwanss Thank you for your work!
Are you going to release the training code?
